pkg/apps/controller/deployer/deployer_controller.go

```diff
@@ -149,6 +150,7 @@ func (c *DeploymentController) handle(deployment *v1.ReplicationController, will
 				return actionableError(fmt.Sprintf("couldn't create deployer pod for %q: %v", deployutil.LabelForDeploymentV1(deployment), err))
 			}
 			updatedAnnotations[deployapi.DeploymentPodAnnotation] = deploymentPod.Name
+			updatedAnnotations[deployapi.DeployerPodCreatedAtAnnotation] = time.Now().Format(time.RFC3339)
```

Comment:
This might get thrown away in case the update at the end fails on 409

Reply:
not lost, on next reconcile we set this back

Reply:
not if the deployer pod is missing (e.g. deleted manually) I think - you set it back when the pod is present only on L182